<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QWidget" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(13, 12, 12, 255), stop:1 rgba(74, 72, 72, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>175</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 89, 11);
color: black;
font-size:22px;
border: none;
border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordConfirmInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>325</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 89, 11);
color: black;
font-size:22px;
border: none;
border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>Confirm password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUp">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>425</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 89, 11);
color: black;
font-size:22px;
border: none;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordInput">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 89, 11);
color: black;
font-size:22px;
border: none;
border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignIn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 147, 43);
color: black;
font-size:14px;
border: none;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
